src/Web/Sfa.Das.Sas.ApplicationServices/Handlers/SitemapHandler.cs

```diff
@@ -54,11 +55,39 @@ public SitemapResponse Handle(SitemapQuery message)
             };
         }
 
+        private IEnumerable<string> GetFrameworkDetailsInSeoFormat()
+        {
+            var frameworks = _getFrameworks.GetAllFrameworks();
+
+            return this.BuildFrameworkSitemap(frameworks);
+        }
+
+        private IEnumerable<string> GetStandardDetailsInSeoFormat()
+        {
+            var standards = _getStandards.GetAllStandards().Where(s => s.IsPublished);
+
+            return this.BuildStandardSitemap(standards);
+        }
+
         private IEnumerable<string> GetProviderDetailsInSeoFormat()
         {
             var providersExcludingEmployerProviders = _getProviders.GetAllProviders().Where(x => x.IsEmployerProvider == false);
-            var identifiers = BuildProviderSitemapFromProviders(providersExcludingEmployerProviders);
-            return identifiers;
+
+            return BuildProviderSitemapFromProviders(providersExcludingEmployerProviders);
+        }
+
+        private IEnumerable<string> BuildFrameworkSitemap(IEnumerable<Framework> frameworks)
+        {
+            return from framework in frameworks
+                   let title = GetTitle(framework)
```

Comment:
Maybe EncodeTitle would be better, I wasn't clear what was going on inline

Reply:
Agreed - makes it a little clearer